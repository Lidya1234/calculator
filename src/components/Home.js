import * as styles from '../style.module.css';

const Home = () => (
  <div className={styles.home}>
    <p>
      Put life in perspective with some short
      yet sage pieces of advice.
      These wise and beautiful words from
      your favorite positive thinkers
      will get you in the right mindset to
      tackle whatever obstacles lie ahead.
      Inspirational quotes from successful
      people like Oprah, Helen Keller, and Maya Angelou can
      help you reframe negative thoughts during tough times
      and change your attitude so you feel
      refreshed and motivated.

    </p>
  </div>
);

export default Home;
