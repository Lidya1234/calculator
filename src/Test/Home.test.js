import React from 'react';
import { render } from '@testing-library/react';

import Home from '../components/Home';
import '@testing-library/jest-dom/extend-expect';

describe('Home has ', () => {
  it('Title of math Quotes', () => {
    const { getByText } = render(<Home />);
    const title = getByText('Math Quotes');
    expect(title).toBeInTheDocument();
  });
  it('same content with the snapshot', () => {
    const { getByText } = render(<Home />);
    const title = getByText('Math Quotes');
    expect(title).toMatchSnapshot();
  });
});
