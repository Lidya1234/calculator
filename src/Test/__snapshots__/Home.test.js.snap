// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home has  Title of math Quotes hvhghj 1`] = `
<h1>
   Math Quotes
</h1>
`;

exports[`Home has  same content with the snapshot 1`] = `
<h1>
   Math Quotes
</h1>
`;
